source/extensions/omni.isaac.lab/omni/isaac/lab/envs/mdp/actions/task_space_actions.py

```diff
@@ -114,6 +114,19 @@ def raw_actions(self) -> torch.Tensor:
     @property
     def processed_actions(self) -> torch.Tensor:
         return self._processed_actions
+    
+    @property
+    def get_jacobian_w(self) -> torch.Tensor:
+        return self._asset.root_physx_view.get_jacobians()[:, self._jacobi_body_idx, :, self._joint_ids]
+    
+    @property
+    def get_jacobian_b(self) -> torch.Tensor:
+        jacobian =  self.get_jacobian_w
+        base_rot = self._asset.data.root_quat_w
+        base_rot_matrix = math_utils.matrix_from_quat(math_utils.quat_inv(base_rot))
+        jacobian[:, :3, :] = torch.bmm(base_rot_matrix, jacobian[:, :3, :])
+        jacobian[:, 3:, :] = torch.bmm(base_rot_matrix, jacobian[:, 3:, :])
+        return jacobian
```

Comment:
Is there a reason to make these properties? If it is only used at one place then you can just keep it there?

Reply:
jtigue suggested that the physx's get_jacobians() is not clear that the returned is in world frame, and keep it there solves the issue but introduced unstated feature, so I instead used property.  I understand that use property in only one place is a bit overkill, and I can change it back if that makes more sense